<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LayMaterials</class>
 <widget class="QWidget" name="LayMaterials">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>336</width>
    <height>507</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LayMaterials</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="2">
    <widget class="QTableView" name="matInfo_tv"/>
   </item>
   <item row="1" column="0">
    <widget class="QPushButton" name="addMat_pb">
     <property name="text">
      <string>Добавить</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="deleteMat_pb">
     <property name="text">
      <string>Удалить</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QPushButton" name="save_pb">
     <property name="text">
      <string>Сохранить</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QPushButton" name="exit_pb">
     <property name="text">
      <string>Выйти</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
